docs/README.md

```diff
@@ -9,7 +9,12 @@ Please refer to [NEURON Source and Release Management Guide](./scm/guide/SCMGuid
 ### Developer documentation 
 
 #### Online
-NEURON consolidated documentation is available at [https://neuronsimulator.github.io/nrn/](https://neuronsimulator.github.io/nrn/):
+Latest NEURON consolidated documentation is available via GithubPages after each merged PR at [https://neuronsimulator.github.io/nrn/](https://neuronsimulator.github.io/nrn/).
```

Comment:
to avoid maintenance burden, should we just remove this stuff? (as a separate PR once RTD is stable)
As GitHub.io only hold latest snapshot, it doesn't bring any value anyway.

Reply:
I don't expect this to change that often.